src/server/cmd/pachctl/cmd/cmd.go

```diff
@@ -157,6 +159,21 @@ This resets the cluster to its initial state.`,
 		Short: "Forward a port on the local machine to pachd. This command blocks.",
 		Long:  "Forward a port on the local machine to pachd. This command blocks.",
 		Run: cmdutil.RunFixedArgs(0, func(args []string) error {
+			pidfile.SetPidfilePath("~/.pachyderm.port.forward.pid")
+			pid, err := pidfile.Read()
+			if err != nil && !os.IsNotExist(err) {
+				return err
+			}
+			if pid != 0 {
+				if err := syscall.Kill(-pid, syscall.SIGKILL); err != nil {
+					if !strings.Contains(err.Error(), "no such process") {
+						return err
```

Comment:
This is making sure we don't error when the process doesn't exist, which is what happens the first time you call `port-forward`. I don't think there's much reason to log that error since you'll always get it the first time you run `port-forward` and there's note much the user would do in response.